### ISSUE TITLE
Can't download file in website

### ISSUE THREAD

**Issue by aokidai** [hlwhl/webview_cef#103]
I can't download any file in web. I try any web but webview not show explorer for choose save local or auto download to download folder. I try in Windows and can't test in other platform.

**Comment by SinyimZhi** [comment]
Generally, applications with inner browser should have their own proxy file download and pop-up file selector capabilities.And it should not be able to load general website but only some website the developer allowed. @aokidai

**Comment by aokidai** [comment]
> Generally, applications with inner browser should have their own proxy file download and pop-up file selector capabilities.And it should not be able to load general website but only some website the developer allowed. @aokidai
So can I help me about issue or send me method for about my issue?

**Comment by aokidai** [comment]
> Generally, applications with inner browser should have their own proxy file download and pop-up file selector capabilities.And it should not be able to load general website but only some website the developer allowed. @aokidai

So can I help me about issue or send me method for about my issue?

**Comment by aokidai** [comment]
> @aokidai find package on pub.dev and combination them is the fun and skills of bulid flutter application。 file_selector will help you have the pop-up file selector capabilities, and download or dio may helps you build the capabilities of download file?

thank you your idea but when I press download button website was called download Api but nothing happened.

**Comment by SinyimZhi** [comment]
@aokidai yes, nothing happened is right caused no click response provided for download button. The innerapp browser generally for load some private domain website or specific website. and you should agreed upon with page developers to use specific JavaScript Channels to transmit download signals and download addresses(Especially some authentication strategies), and implement download logic outside of this plugin. If you want to load some public and well-known websites and download files or something else, why don't you just call up the default browser program for the operating system to load them.

**Comment by aokidai** [comment]
> The innerapp browser generally for load some private domain website or specific website. and you should agreed upon with page developers to use specific JavaScript Channels to transmit download signals and download addresses(Especially some authentication strategies), and implement download logic outside of this plugin. If you want to load some public and well-known websites and download files or something else, why don't you just call up the default browser program for the operating system to load them.

Thank you very much!

**Comment by hlwhl** [comment]
Closing this as not planned, consistent with the existing `wontfix` decision. The plugin does not currently implement CEF download-handler integration or a native save dialog. Applications can handle downloads explicitly through a JavaScript channel and a Flutter download/file-selector implementation, or open unsupported public download flows in the system browser.
